settings/browsersettings.cpp

```diff
@@ -13,7 +13,9 @@
 #include <QXmlStreamReader>
 #include "browsersettings.h"
 
-static const QString openSearchPath("/usr/lib/mozembedlite/chrome/embedlite/content/");
+static const QStringList gOpenSearchPathList = QStringList() \
+                                            << QString("/usr/lib/mozembedlite/chrome/embedlite/content/") \
+                                            << QDir::homePath() + "/.local/share/org.sailfishos/sailfish-browser/opensearch/";
```

Comment:
Please move ```/usr/lib/mozembedlite/chrome/embedlite/content/``` to a define in common.pri (similar to DEFAULT_COMPONENTS_PATH) and replace also from browsersettings.cpp. For user specific search engine paths use 
QStandardPaths::writableLocation(QStandardPaths::DataLocation) + "/searchEngines".